content/en/docs/sdk/gosdk.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Introduction"
+description: "Introduces the Helm Go SDK"
+aliases: ["/docs/gosdk"]
+weight: 1
+---
+Helm's Go SDK enables custom software to leverage Helm charts and Helm's functionality for managing Kubernetes software deployment (In fact, the Helm CLI is effectively just one such tool!)
```

Comment:
I think we should integrate with this existing Go SDK section under Advanced Helm Techniques https://helm.sh/docs/topics/advanced/#go-sdk

Reply:
For some reason I didn't do this initially (I can't remember why).

I think the SDK deserves its own standalone section. I'll move some/most of the content from `https://helm.sh/docs/topics/advanced/#go-sdk` into this new SDK section and replace with a link

Reply:
done